index.js

```diff
@@ -181,25 +199,34 @@ class Markdown extends Component {
              */
             delete extras.blockQuote;
         }
-        const nodes = this.renderNodes(node.props.children, key, extras);
+        const children = this.renderNodes(node.props.children, key, extras);
 
         if (isBlockQuote) {
             style.push(styles.blockQuote)
+            if (this.props.renderBlockQuote) {
+                return this.props.renderBlockQuote(children)
+            }
             return (
                 <View key={'blockQuote_' + key} style={[styles.block, styles.blockQuote]}>
-                    <Text>{nodes}</Text>
+                    <Text>{children}</Text>
                 </View>
             );
         }
-        else if (Utils.isTextOnly(nodes)) {
+        else if (Utils.isTextOnly(children)) {
+            if (this.props.renderBlockText) {
+                return this.props.renderBlockText(children)
+            }
             return (
-                <Text key={`block_text_` + key} style={styles.block}>{nodes}</Text>
+                <Text key={`block_text_` + key} style={styles.block}>{children}</Text>
             );
         }
         else {
+            if (this.props.renderBlock) {
+                return this.props.renderBlock(children);
+            }
             return (
                 <View key={'block_' + key} style={styles.block}>
-                    {nodes}
+                    {children}
```

Comment:
Would it be better to expose only a `renderBlock` and let them know if quoted, text, or plain? Unsure of the differences between text and plain block tbh